.gitignore

```diff
@@ -33,8 +32,20 @@ taoTestTaker/
 taoTests/
 taoWfAdvTest/
 taoWfTest/
-vendor/
-wfAuthoring/
-wfEngine/
 taoOutcomeRds/
 taoOutcomeUi/
+pciSamples/
```

Comment:
A whitelist would be more safe and maintainable. After a TAO install I have extensions, like `taoDeliverySchedule` which are not on the list.

Reply:
@jbout should be fixed into develop later